message-me/app/javascript/packs/application.js

```diff
@@ -12,3 +12,6 @@ import '@doabit/semantic-ui-sass'
 Rails.start()
 Turbolinks.start()
 ActiveStorage.start()
+$(document).on('turbolinks:load', function() {
```

Comment:
Can you comment this block out and take a screenshot of the page?

Reply:
Before
![image](https://user-images.githubusercontent.com/67599130/126303368-a222804f-4a49-4626-bbf1-beaca74a9ea1.png)
After
![image](https://user-images.githubusercontent.com/67599130/126303420-64ba7b9d-2318-4618-856b-080f277811ed.png)

After adding only the dropdown works

Reply:
i ask for a screenshot of the page.
Where is it? 
Didn't I keep tell you not to assume? there are other things I want to see. Not this.

Reply:
When commented out
![image](https://user-images.githubusercontent.com/67599130/126316663-0f23dee6-7087-4516-950f-1c5750ca035b.png)